README.md

```diff
@@ -183,6 +183,21 @@ Pylance provides users with the ability to customize their Python language suppo
         -   `true`
         -   `false` (default)
 
+-    `python.analysis.inlayHints.pytestParameters`
+    -    Enable/disable inlay hints for pytest function parameters.
+    ```python
+    def test_foo(my_fixture):
+        assert(myfixture)
```

Comment:
Maybe remove the second line (assert) or simplify to `pass` or `...` since it seems unrelated to what you're explaining here?

Reply:
Good idea

Reply:
If this is already in the docs/package.json, just ignore this I guess.

Reply:
or def test_foo(my_fixture): ... to make it 1 line